troubleshoot/index.md

```diff
@@ -57,14 +57,48 @@ If you have an [Elastic subscription](https://www.elastic.co/pricing), you can c
   If you contact us by email, use the email address you registered with so we can help you more quickly. If your registered email is a distribution list, you can register a second email address with us. Just open a case to let us know the name and email address you want to add.
   :::
 
+  :::{warning}
+  All cases opened by email default to normal severity. For incidents, open a case through the [Elastic Support Portal](https://support.elastic.co/) and select the [appropriate severity](https://www.elastic.co/support/welcome#what-to-say-in-a-case). 
+  :::
+
 ## Working with support [troubleshoot-work-with-support]
 
-Try these tips when opening a support case:
+When you open a support case:
+
+* Describe the problem and its [contextual severity](https://www.elastic.co/support/welcome#what-to-say-in-a-case). 
+
+  * If an error message was encountered, include full error message and timezone-designated dates and times of when the problem occurred.
+
+  * If the problem is UI-related and can't be replicated using API calls, pull a [browser network log](https://www.elastic.co/blog/generating-browser-har-file-kibana-troubleshooting). 
+
+* Upload the related product's diagnostics and debug logs:
+
+  * If hosting on {{serverless-full}} or {{ech}}, Support can pull diagnostics on your behalf as long as you include resource IDs:
```

Comment:
```suggestion
  * If you're using {{serverless-full}} or {{ech}}, Support can pull diagnostics on your behalf as long as you include resource IDs:
```
Just avoiding repitiion since "hosted" is part of the ECH product name.